.github/workflows/pr.yaml

```diff
@@ -0,0 +1,50 @@
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review, edited, closed]
```

Comment:
I think we can remove closed here and add `reopened`
```
[opened, reopened , synchronize, ready_for_review, edited]
```

Reply:
done ✅